### PR TITLE
[13.0][MIG] packaging_uom

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

CC @ForgeFlow

### CODE REVIEW

File: packaging_uom/models/product_packaging.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    @api.constrains("uom_id")
+    def _check_uom_id(self):
+        """ Check uom_id is not null
+
+        Since the field can be computed by the inverse method on 'qty',
+        it's no more possible to add a sql constrains on the column uom_id.
+        """
+        for rec in self:
+            if not rec.uom_id:
+                raise ValidationError(_("The field Unit of Measure is " "required"))
```
Comment (rousseldenis) [inline packaging_uom/models/product_packaging.py]: Maybe a typo generated by black

**Review by rousseldenis — APPROVED** [review]
Code review

File: .isort.cfg [hunk 2]
```diff
@@ -1,2 +1,12 @@
 [settings]
-known_third_party = odoo,psycopg2,setuptools
```
Comment (emagdalenaC2i) [inline .isort.cfg]: Are you sure to remove odoo here?

File: packaging_uom/readme/HISTORY.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+11.0.1.0.0 (2018-08-28)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* [MIG] packaging_uom: Migration to 11.0
+
+12.0.1.0.0 (2019-05-27)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* [MIG] packaging_uom: Migration to 12.0
```
Comment (emagdalenaC2i) [inline packaging_uom/readme/HISTORY.rst:19]: I don't know if we should keep this file but you could add the new migration here

File: packaging_uom/readme/INSTALL.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
+To install this module, you need to:
+
+* Click on install button
```
Comment (emagdalenaC2i) [inline packaging_uom/readme/INSTALL.rst]: This file has nonsense IMHO

File: product_code_unique/tests/test_code_unique.py [hunk 5]
```diff
@@ -1,6 +1,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (emagdalenaC2i) [inline product_code_unique/tests/test_code_unique.py]: This file don't belong to this module

**Review by emagdalenaC2i — CHANGES_REQUESTED** [review]

**Comment by emagdalenaC2i** [comment]
@HviorForgeFlow and @rousseldenis Please, could you review #570 ?

File: .isort.cfg (hunk 2 above)
Comment (HviorForgeFlow) [reply .isort.cfg]: I've just copied from another repository where CI is green

File: product_code_unique/tests/test_code_unique.py (hunk 5 above)
Comment (HviorForgeFlow) [reply product_code_unique/tests/test_code_unique.py]: It fixes CI

Comment (rousseldenis) [reply product_code_unique/tests/test_code_unique.py]: @HviorForgeFlow Could you do it in another PR or at least in another commit ? Thanks

Comment (HviorForgeFlow) [reply product_code_unique/tests/test_code_unique.py]: Sure! @rousseldenis

**Review by emagdalenaC2i — APPROVED** [review]

**Pull request merged**